docs/buildah-build.1.md

```diff
@@ -497,6 +497,10 @@ Path to an alternative .containerignore (.dockerignore) file.
 Write the built image's ID to the file.  When `--platform` is specified more
 than once, attempting to use this option will trigger an error.
 
+**--inherit-labels** *bool-value*
+
+Allows users to specify if they want to inherit labels from base image or not. (Default is `true`).
```

Comment:
"Allows users" is kind of implied here, in the man page.  Describe what the tool will do when the flag is specified.

Reply:
Perhaps

Inherit the labels from the base image (default true).

I'm assuming the default is true; I haven't read the rest.